API/Services/ReadingListService.cs

```diff
@@ -529,7 +537,7 @@ public async Task CreateReadingListsFromSeries(Series series, Library library)
                 arcNumbers[i] = int.MaxValue.ToString();
             }
             if (string.IsNullOrEmpty(arcs[i]) || !int.TryParse(arcNumbers[i], out _)) continue;
-            data.Add(new Tuple<string, string>(arcs[i], arcNumbers[i]));
+            data.Add(new Tuple<string, string>(arcs[i].Trim(), arcNumbers[i]));
```

Comment:
Move arcs[i] into a variable and trim there, so you can use the !string.IsNullOrEmpty(arcs[i]) if statement appropriately.

Reply:
Probably easier to switch is IsNullOrWhiteSpace as well.

Reply:
Reviewing the .net documentation has indicated a cleaner approach that I switched to. Using the Split to remove whitespaces and empty entries.